inlong-manager/manager-plugins/src/main/java/org/apache/inlong/manager/plugin/listener/RestartSortListener.java

```diff
@@ -73,15 +73,15 @@ public ListenerResult listen(WorkflowContext context) throws Exception {
                 new TypeReference<Map<String, String>>(){});
         kvConf.putAll(result);
         if (StringUtils.isEmpty(kvConf.get(InlongGroupSettings.SORT_JOB_ID))) {
-            String message = String.format("inlongGroupId:%s not add restartProcess listener,SORT_JOB_ID is empty",
+            String message = String.format("inlongGroupId:%s not add restartProcess listener, SORT_JOB_ID is empty",
```

Comment:
Change to `"groupId [%s] not add restartProcess listener, as the sortJobId is empty"`.

Reply:
ok done